browser/src/control/jsdialog/Util.Dropdown.js

```diff
@@ -182,6 +193,8 @@ JSDialog.OpenDropdown = function (id, popupParent, entries, innerCallback, popup
 
 			if (eventType === 'selected')
 				JSDialog.CloseDropdown(id);
+			else if (eventType === 'render_entry')
```

Comment:
no need to reimplement anything, it will be handled by the same code as original widget